### ISSUE TITLE
OperationFilter issue

### ISSUE THREAD

**Issue by doneal1992** [domaindrivendev/Swashbuckle#252]
public class SwaggerConfig
    {
        public static void Register()
        {
            var thisAssembly = typeof(SwaggerConfig).Assembly;

            GlobalConfiguration.Configuration 
                .EnableSwagger(c =>
                    {
                        
                        c.SingleApiVersion("v1", "SearchDms");
                       // c.OperationFilter<AddParametersFilter>();
                        
                       
                    })
                .EnableSwaggerUi(c =>
                    {
                        
                    });

           
        }
    }
}

if the c.OperationFilter<AddParametersFilter>(); is commented out my api will load, but as soon as I try to add it into the code, when the swagger ui loads i get : 

500 : undefined http://local-search.ignitionone.com:80/swagger/docs/v1

What would be causing this? I added the [Authorize] attribute to one of my controllers to see if I could get at least one of them to show however no luck so far.

**Comment by domaindrivendev** [comment]
You'll need to provide some more information. Can you navigate directly to the raw swagger URL (e.g. the one you provided above) and let me know what error message it's showing?

**Comment by doneal1992** [comment]
I actually figured it out. In my AddParameterFilter class I had to encompass operation.parameter.add with a try catch block that caught nullreferences. it worked after that. However, not that I have commited my project to source control and merged my branch with another. The ui has stopped retrieving my swagger docs. If i open the google developer console it seems to be failing and underscore.map-min
